NebulaPatcher/Patches/Dynamic/UIStarmap_Patch.cs

```diff
@@ -0,0 +1,27 @@
+﻿using HarmonyLib;
+using NebulaWorld;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```

Comment:
These dependencies are not used here, right?

Reply:
Done